src/locales/zh.json

```
@@ -813,6 +813,19 @@
         "rgb++": "RGB++",
         "unnamed": "未命名"
       },
+      "tags_description": {
+        "invalid": "此Tag标记的资产名称中含有无效字符。",
+        "suspicious": "此Tag标记的资产名称中含有大量隐藏字符类型。",
+        "out-of-length-range": "此Tag标记的资产名称长度超过60字符。",
+        "duplicate": "区分大小写前提下，此Tag标记的资产名称与已发行过的资产重复。",
+        "rgb++": "此Tag标记的资产是RGB++ 协议兼容资产，可以在BTC 和CKB之间相互转移。",
+        "layer-1-asset": "此Tag标记的资产是发行在Layer1 上的资产。",
+        "layer-2-asset": "此Tag标记的资产是发行在Layer2 上的资产。",
+        "verified-on": "此Tag标记的资产是经过Cell Studio 校验的资产。",
```

```suggestion
        "invalid": "此 Tag 标记的资产名称中含有无效字符。",
        "suspicious": "此 Tag 标记的资产名称中含有大量隐藏字符类型。",
        "out-of-length-range": "此 Tag 标记的资产名称长度超过 60 字符。",
        "duplicate": "区分大小写前提下，此 Tag 标记的资产名称与已发行过的资产重复。",
        "rgb++": "此 Tag 标记的资产是 RGB++ 协议兼容资产，可以在 BTC 和 CKB 之间相互转移。",
        "layer-1-asset": "此 Tag 标记的资产是发行在 Layer1 上的资产。",
        "layer-2-asset": "此 Tag 标记的资产是发行在 Layer2 上的资产。",
        "verified-on": "此 Tag 标记的资产是经过 Cell Studio 校验的资产。",
```
Add whitespace between around English word/number